module/localisation/FieldLocalisationNLopt/README.md

```diff
@@ -0,0 +1,24 @@
+# FieldLocalisationNLopt
+
+## Description
+
+A localisation method for estimating the where the field is in world space, which relies on field line points and
+field line intersections.
```

Comment:
it might be useful to have some information on the NL optimisation side of things.